Backend/internal/auth/logout.go

```diff
@@ -0,0 +1,52 @@
+// TITLE: Login functions
+// Created by (Jacky: FafnirZ) (09/21)
+// Last modified by (Jacky: FafnirZ)(12/09/21)
+// # # #
+/*
+Logout functionalities
+**/
+package auth
+
+import (
+	_httpUtil "DiffSync/httpUtil"
+	_session "DiffSync/internal/session"
+	"log"
+
+	"net/http"
+)
+
+// expecting a post request with body of form data
+// expecting header to contain session-token
+// will perform the redirection in frontend
+// backend's job for logout is only to remove the HTTPONLY cookie
+
+func LogoutHandler(w http.ResponseWriter, r *http.Request) {
+
+	switch r.Method {
+	case "POST":
+		authenticated, _ := _session.IsAuthenticated(w, r)
+		log.Print(authenticated)
+		if authenticated {
+			// CORS headers
+			w.Header().Set("Access-Control-Allow-Origin", "http://localhost:3001")
```

Comment:
super small nitpick but maybe make this a constant just so that if we deploy this then we dont need to go through all change all instances of localhost

Reply:
yeah truee, I was thinking maybe we could use an environment variable maybe??

Reply:
yah env variable sounds good coz we can also pass it in with docker